src/librustdoc/html/render/context.rs

```diff
@@ -129,6 +129,9 @@ crate struct SharedContext<'tcx> {
     crate cache: Cache,
 
     crate call_locations: AllCallLocations,
+    /// The key is the notable trait text and the value is its index (we add it in the DOM so the
+    /// JS knows which notable trait to pick).
+    crate notable_traits: RefCell<FxHashMap<(String, String), usize>>,
```

Comment:
Are you absolutely sure this needs a RefCell?

Reply:
For now at least it does unfortunately...

Reply:
Ah, I didn't notice this is in the SharedContext.